configuration/configmgr/configmgrlib/mod_template_support/Operation.cpp

```diff
@@ -180,3 +191,35 @@ void Operation::saveAttributeValues(Variables *pInputs, const std::shared_ptr<En
         }
     }
 }
+
+
+void Operation::processNodeValue(Variables *pVariables, const std::shared_ptr<EnvironmentNode> &pEnvNode)
+{
+    if (m_nodeValueValid)
```

Comment:
Should you do anything if m_nodeValueValid == false?

Reply:
No. This flag is only true if the template is attempting to change the value of an element, not an attribute value of an element. So, if false, no further consideration.